README.md

```diff
@@ -285,15 +220,7 @@ You can find an ever growing list at [Awesome IPFS](https://awesome.ipfs.io/apps
 
 ### Other Community Resources
```

Comment:
This section title doesn't make sense here since this is still part of the repo directory and community is the next section
```suggestion
```